core/spec/models/spree/tax_rate_spec.rb

```diff
@@ -402,7 +401,7 @@
     end
 
     context 'not taxable line item ' do
-      let!(:line_item) { @order.contents.add(@nontaxable.master, 1) }
+      let!(:line_item) { Spree::AddItemToCart.new.call(order: @order, variant: @nontaxable.master).value }
```

Comment:
I suppose we can think about a way to move these things into factories, what do you think?